packages/server/src/utils/gpu-setup.ts

```diff
@@ -0,0 +1,9 @@
+import { docker } from '../constants';
+
+export async function setupGPUSupport() {
+  await docker.swarmUpdate({
+    TaskDefaults: {
+      GenericResources: [{ DiscreteResourceSpec: { Kind: 'gpu', Value: 1 } }]
+    }
+  });
+}
```

Comment:
This code is only for dokploy server not for remote, since the docker object is referring to the local docker from dokploy server

Reply:
you should use this function and pass a optional serverId

https://github.com/Dokploy/dokploy/blob/canary/packages/server/src/utils/servers/remote-docker.ts

Reply:
Remove unused imports and interfaces in the next commit, I missed this part to fix